googling.md

```diff
@@ -0,0 +1,22 @@
+# question number one
+  
+  
+  Phrase 1: how can change a typo in commit messages?
+
+  Phrase 2: how can i change a commit messages before I push it to github?
+
+  # question number two
+   Phrase 1: what are the Ruby error  messages with meaning?
+
+  Phrase 2: how can i fix a string error message in Ruby program?
+
+  # question number three
+
+ Phrase 1: how can I print a git log  in oneline?
+   
+ Phrase 2: how can i git log in a mod 0?
```

Comment:
I'd say the mod0 program is irrelevant in this case. You'd be better off including the keyword `git` like `git log in oneline`.